src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,52 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    if (!Animal.alive.includes(this)) {
+      Animal.alive.push(this);
```

Comment:
There's no need to check if instance is already in the 'alive' array. Each animal is created only once and immediately added to the array.

Reply:
I had some issues with Lint and it was one of the attempts